README.md

```diff
@@ -2,3 +2,71 @@ FIG Cookies
 ===========
 
 Managing Cookies for PSR-7 Requests and Responses.
+
+# Accessing cookies from Request
+
+```php
+use Dflydev\FigCookies\Cookies;
+
+// $request = a psr7request;
+$cookies = Cookies::fromRequest($request);
+```
+
+## Check a cookie exists
+
+You can check whether a cookie by name exists via `has` method which returns `bool`.
+
+```php
+$cookies->has('name');
+```
+
+## Get cookie
+
+You can get a `Cookie` object via `get` method.
+
+```php
+$cookie = $cookies->get('name');
+```
+
+This will return a `Dflydev\FigCookies\Cookie` object via which you can get
+name, value etc via `getName()` and `getValue()` respectively.
+
+## Change cookie value
+
+You can change the value of a cookie via `withValue` method.
+
+```php
+$cookie->withValue('some value');
```

Comment:
Also, it should be noted that simply changing a cookie value doesn't update values in cookies, either. The two ways to "change (or set) a cookie" would probably be two options like this:

```php
// Get the "token" cookie and set a new value.
$cookie = $cookies->get('token')->withValue('some new value');

// Replace the "token" cookie in the collection of Cookies.
$cookies = $cookies->with($cookie);
```

... or...

```php
// Create a "token" cookie and set a new value.
$cookie = Cookie::create('token')->withValue('some value');

// Set the "token" cookie in the collection of Cookies.
$cookies = $cookies->with($cookie);
```